test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -701,7 +702,20 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 				}
 				kernelParameters := []string{noHzParam, "tsc=nowatchdog", "nosoftlockup", "nmi_watchdog=0", "mce=off", "skew_tick=1",
 					"processor.max_cstate=1", "intel_idle.max_cstate=0", "intel_pstate=disable", "idle=poll"}
-				checkTunedParameters(workerRTNodes, stalldEnabled, sysctlMap, kernelParameters, rtKernel)
+				for i := 0; i < len(workerRTNodes); i++ {
+					node := &workerRTNodes[i]
+					By("Waiting for TuneD to start")
+					pod, err := utilstuned.GetPod(context.TODO(), node)
+					Expect(err).ToNot(HaveOccurred())
+					cmd := []string{"test", "-e", "/run/tuned/tuned.pid"}
+					Expect(util.WaitForCmdInPod(5*time.Second, 5*time.Minute, pod, cmd...)).ShouldNot(BeEmpty())
```

Comment:
this works but is done serially, so adds time to already long running tests. We should parallelize the wait **in a future improvement PR**. For now let's move forward, it's more important to reduce the flakes

Reply:
Sounds like a great idea.

Reply:
I'll start working on that once this one gets merged

Reply:
I should have noticed this earlier, and we discussed this offline: let's add this code (moved in a helper or so) into all the `When` blocks pertaining to `Context("WorkloadHints")`

Reply:
Done. Moved this logic into all the relevant tests` instances